tf-build-tools/tf-test/src/main/groovy/com/deliveredtechnologies/terraform/fluent/api/TfState.groovy

```diff
@@ -0,0 +1,14 @@
+package com.deliveredtechnologies.terraform.fluent.api
+
+import groovy.json.JsonSlurper
+
+class TfState {
+
+    Map rawState
+
+    JsonSlurper slurper = new JsonSlurper()
```

Comment:
Is it possible to make this as a method variable in the constructor, so that there is a possibility of reclaim the resources after successful slurp?